extensions/amp-analytics/0.1/analytics-root.js

```diff
@@ -211,11 +211,19 @@ export class AnalyticsRoot {
       let result = null;
       // Query search based on the selection method.
       if (selectionMethod == 'scope') {
-        found = scopedQuerySelector(context, selector);
+        try {
+          found = scopedQuerySelector(context, selector);
+        } catch (e) {
+          user().error(TAG, 'Invalid query selector :', selector, e);
```

Comment:
We also throw user error when can't find target element.  I figure invalid selector is somehow different from that error. 
Let me know if two errors "invalid selector" + "element not found" are not wanted..